programs/system/CancelOrder.go

```diff
@@ -0,0 +1,227 @@
+// Copyright 2022 bloXroute-Labs
```

Comment:
Convention in the rest of the repo is to do underscore case for names, like `cancel_order.go`.

Reply:
Oh wait it's not consistent?

Reply:
Hm, this also seems like the wrong place for this file `system/` seems to refer to the Solana `system` program, which kind is responsible for all the high level stuff that happens in Solana –– like managing program ownership of other accounts and things like that. So I don't think this Serum specific instruction belongs here? But then do you need access to this module to implement the instruction properly?

Reply:
Also, I'd prefer we not add copyright bloXroute-Labs to the file –– in the ideal case we'd get this change merged into `solana-go` main branch once we get some feedback on intended usage.